geoutils/raster/delayed.py

```diff
@@ -0,0 +1,777 @@
+"""
+Module for dask-delayed functions for out-of-memory raster operations.
+"""
+from __future__ import annotations
+
+import warnings
+from typing import Any, Literal, TypeVar
+
+import dask.array as da
+import dask.delayed
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import rasterio as rio
+from scipy.interpolate import interpn
+
+from geoutils._typing import NDArrayNum
+from geoutils.projtools import _get_bounds_projected, _get_footprint_projected
+
+# 1/ SUBSAMPLING
+# Getting an exact subsample size out-of-memory only for valid values is not supported directly by Dask/Xarray
```

Comment:
Maybe mention "not supported as of 30/04/2024" or "as of version xxx" ? Here and also for the other items below.

Reply:
Added